sections/appendix/testplan.tex

```diff
@@ -0,0 +1,48 @@
+\section{Test plan}
+
+\subsection{Documentation}
+All functions should be documented. 
+This is done by writing comments in the code following the standards of the language.
+\\\\
+For C\#: 
+\\
+\url{https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/language-specification/documentation-comments}
+\\\\
+For Python:
+\\
+\url{https://www.geeksforgeeks.org/python-docstrings/}
+\\\\
+These comments should explain what the parameters of the function are and a short summary of what the function does.
+\\\\
+If the function that is being implemented deals with more complex concepts, additional comments should be written in the code to explain what is happening in the function.
+
+\subsection{Testing}
+\subsubsection{What to test}
+All code that can influence the user's experience should be tested. 
+This means that every function that would influence the program negatively, if it had an incorrect calculation should be tested to try to avoid this.
```

Comment:
```suggestion
This means that every function that would influence the program negatively if it had an incorrect calculation should be tested to try to avoid this.
```